src/combobox/combobox.component.ts

```diff
@@ -454,6 +474,18 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 		});
 	}
 
+	closeClearButton(event) {
```

Comment:
Sorry, going to be slightly picky ... `clearInput` or something similar may make more sense ... since we're not closing anything

Reply:
Okay!